SDALib-CPP/SDALib-CPP/SDA_portability.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "ipclib_portability.h"
+
+#ifdef WIN32
+#define SDA_IP_TYPE PCWSTR
+#define LOCALHOST   L"127.0.0.1"
+
+#define SDA_THROW_EXCEPTION(p_msg)     \
+    std::cerr << (p_msg) << std::endl; \
+    throw std::exception(p_msg)
+#endif
+
+#ifdef __linux__
+#define LOCALHOST   "127.0.0.1"
+#define SDA_IP_TYPE const char*
+#define SDA_THROW_EXCEPTION(p_msg)     \
+    std::cerr << (p_msg) << std::endl; \
+    throw std::exception();
```

Comment:
Can an exception not contain a message on linux?

Reply:
no